### DISCUSSION TITLE
How does Zettlr handles data that was NOT edited or created by it?

### DISCUSSION THREAD

**Discussion by scotrod** [Zettlr/Zettlr#4676] [Q&A]
Hey All. Since Zettlr does not provide any kind of synchronization capabilities or smartphone application, I've set up Zettlr's workspace to be synced via syncthing over to my remote machines and smartphone. 

I wanted to open this discussion to ask around if changes, or new files which were not created by Zettlr itself may bring some issues or conflicts. I've got a backup of my notes, but since Zettlr is going to become my main note-taking application, I want to be quite sure that my data is safe with it.

Thanks!

**Comment by nathanlesage** [comment]
Let's do a quick overview:

> How does Zettlr handles data that was NOT edited or created by it?

Short answer: Sometimes well, sometimes awfully. The main reason for this is that in the past I've made some really dumb infrastructural decisions as I've basically "learned on the job" (I didn't know any JavaScript six years ago when I started, now I know much more, but it takes time to replace the brittle code with more solid code, similarly to renovating a house). Fixing many of these brittly issues is my main agenda right now because of all the issues that you've seen yourself on the issue tracker.

But now, with all this hassle, what can _you_ do to ensure that _your_ files are always safe until that happens?

* I've seen over the years that especially MS OneDrive appears to be really problematic with weird modtime things that make Zettlr's logic choke. I'm running a Nextcloud instance for 6 years with zero problems whatsoever. Other users have reported that Dropbox and Google Drive appears to be working well, just as some other minor cloud services that I do not remember off the top of my head.
* Disable autosave. I have an idea what might be happening, but it will take a few weeks until I get to fixing that one.
* Don't edit files outside Zettlr and inside at the same time. This can sometimes upset the logic.

And that's really it. I can vouch that this behavior will not lead to any data loss (I'm exclusively using Zettlr, and I haven't had data loss in years, but I am also a little bit wary, just because I know what stupidity I did seven years ago …). I understand that this might not be the answer you were hoping for, but I'm striving to do better with the limited time I have!

**Comment by ghost** [comment]
I am not sure but it looks like Zettlr don't have text-wrapping settings. As I noticed, if you saved your file with hard-wrapping (from another program), Zettlr will show it a bit differently than it was shown when text was without hard-wrapping.
